app/components/forms/wizard/sessions-speakers-step.js

```diff
@@ -86,6 +86,28 @@ export default Component.extend(EventWizardMixin, FormMixin, {
               prompt : this.l10n.t('Please give an end time')
             }
           ]
+        },
+        microlocation: {
+          identifier : 'microlocation',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter name for microlocation')
+            }
+          ]
+        },
+        microlocationFloor: {
+          identifier : 'microlocation_floor',
+          rules      : [
+            {
+              type   : 'empty',
```

Comment:
No need for this then !

Reply:
Its not working without empty. 

![Screenshot from 2020-01-22 17-23-52](https://user-images.githubusercontent.com/46647141/72892247-118ce580-3d3c-11ea-8667-6888d2920b24.png)

Reply:
@kushthedude  `tax_rate` is also implemented as this. 

![Screenshot from 2020-01-22 17-26-14](https://user-images.githubusercontent.com/46647141/72892379-59ac0800-3d3c-11ea-9eee-a6f7bf7453b0.png)